docs/_source/components-overview/pop_syn/population_params.rst

```diff
@@ -377,69 +630,165 @@ These parameters contain options on how the population is evolved, in practical
-   * - `#orbital_separation_min`
-     - 5.0 (float i [0,inf])
-   * - `#orbital_separation_max`
-     - 1e5 (float i [0,inf])
-   * - `#log_orbital_separation_mean`
-     - None (float i [0,inf] used only for orbital_separation_scheme ='log_normal')
-   * - `#log_orbital_separation_sigma`
-     - None (float i [0,inf] used only for orbital_separation_scheme ='log_normal')
-   * - `eccentricity_scheme`
-     - 'zero' (options are 'zero', 'thermal', 'uniform')
+  * - Parameter
+    - Description
+    - Default Value
+
+  * - ``optimize_ram``
+    - | Save population in batches.
+    - ``True``
+
+  * - ``ram_per_cpu``
+    - | Set maximum RAM per CPU before batch saving (in GB).
+    - ``None``
+
+  * - ``dump_rate``
+    - | Batch save after evolving N binaries.
+    - ``2000``
+
+  * - ``temp_directory``
+    - | Folder for keeping batch files.
+    - ``'batches'``
+
+  * - ``tqdm``
+    - | Enable progress bar.
+    - ``False``
+
+  * - ``breakdown_to_df``
+    - | Convert BinaryStars into DataFrames after evolution.
+    - ``True``
+
+  * - ``use_MPI``
+    - | If True, evolve with MPI (equivalent to: ``from mpi4py import MPI, comm = MPI.COMM_WORLD``).
+    - ``True``
+
+  * - ``metallicity``
+    - | In units of solar metallicity.
+    - ``[2., 1., 0.45, 0.2, 0.1, 0.01, 0.001, 0.0001]``
+
+  * - ``entropy``
+    - | Random Number Generation: uses system entropy.
+    - ``None``
+
+  * - ``number_of_binaries``
+    - | Number of binaries to evolve.
+    - ``1000000``
+
+  * - ``star_formation``
+    - | What star formation is used to sample the binaries.
+      | Options:
+    
+      * ``'constant'``: sample with a constant rate over time
+      * ``'burst'``: sample from a burst of star formation
+      * ``'custom_linear'``: sample with a custom linear rate over time
+      * ``'custom_log10'``: sample with a custom log10 rate over time
+      * ``'custom_linear_histogram'``: sample from a custom linear histogram rate over time
+      * ``'custom_log10_histogram'``: sample from a custom log10 histogram rate over time
+    - ``'burst'``
+
+  * - ``max_simulation_time``
+    - | Maximum simulation time (in years).
+    - ``13.8e9``
+
+  * - ``binary_fraction``
+    - | Fraction of binaries (0 < fraction <= 1).
+    - ``1``
+
+  * - ``primary_mass_scheme``
+    - | Options:
+    
+      * ``Salpeter``: `Salpeter E. E., 1955, ApJ, 121, 161 <https://ui.adsabs.harvard.edu/abs/1955ApJ...121..161S/abstract>`_
+      * ``'Kroupa1993'``: `Kroupa P., Tout C. A., Gilmore G., 1993, MNRAS, 262, 545 <https://ui.adsabs.harvard.edu/abs/1993MNRAS.262..545K/abstract>`_
+      * ``'Kroupa2001'``: `Kroupa P., 2001, MNRAS, 322, 231 <https://ui.adsabs.harvard.edu/abs/2001MNRAS.322..231K/abstract>`_
+    - ``'Kroupa2001'``
+
+  * - ``primary_mass_min``
+    - | Minimum primary mass (in solar masses)
+      | limits: 0-300
+    - ``7``
+
+  * - ``primary_mass_max``
+    - | Maximum primary mass (in solar masses).
```

Comment:
Should we here tell about restrictions, e.g. that the minimum<maximum? Please check, how the code is doing this checking exactly.